Nikita_Korzhavin/js10myBind.js

```diff
@@ -0,0 +1,26 @@
+Function.prototype.myBind = function(context) {
+  if (typeof this !== 'function') {
```

Comment:
It seems that you don't need this check since you are changing the prototype of Function. If you will try to invoke `myBind` from string, for instance, you'll get `TypeError: myBind is no a function`

Reply:
fixed